samples/Store/Integration/AutoDataAttribute.fs

```diff
@@ -2,6 +2,22 @@
 
 open System
 
+open Domain
+open FsCheck.FSharp
+open FSharp.UMX
+
+module Arb =
+    let generate<'t> = ArbMap.defaults |> ArbMap.generate<'t>
```

Comment:
was asking for a blank L10 but this is consistent w L13 so nmm!
Also should name of helper be ArbMap?
Also should helper be called `defaultGen`
Also should we log a request for FsCheck to put this in the box ?

Reply:
https://github.com/search?q=repo%3Afscheck%2FFsCheck%20ArbMap&type=code
/ https://github.com/fscheck/FsCheck/blob/bf0d5707ba7590e83e51d72329e0ef861f2414fc/src/FsCheck/FSharp.ArbMap.fs#L49-L50

suggests `ArbMap.defaultsGenerate`, `ArbMap.defaultGenerator` or `defaultGenFor` might be a better name?

(And I like `defGen`!)